deepchem/feat/__init__.py

```diff
@@ -34,7 +34,8 @@
 from deepchem.feat.molecule_featurizers import RDKitDescriptors
 from deepchem.feat.molecule_featurizers import SmilesToImage
 from deepchem.feat.molecule_featurizers import SmilesToSeq, create_char_to_idx
-from deepchem.feat.molecule_featurizers import MATFeaturizer
+from deepchem.feat.molecule_featurizers import RobertaFeaturizer
+from deepchem.feat.molecule_featurizers import MATFeaturizers
```

Comment:
Is there a reason why we changed MATFeaturizer to MATFeaturizers here?

Reply:
no - must be a typo, good catch!